A2-service-configs-in-dns.md

```diff
@@ -55,9 +55,13 @@ determine which choice will be selected by a given client:
     // "python", etc).  Each string is case insensitive.
     "clientLanguage": [string],
     // Percentage: integer from 0 to 100 indicating the percentage of
-    // clients that should use this choice.  If present, the number must
-    // match the regular expression `^0|[0-9]|[1-9][0-9]|100$`
-    // All other numbers are considered invalid.
+    // clients that should use this choice.  Each client's resolver should
```

Comment:
Just to be clear:   If the channel goes into idle mode, shuts down the resolver, and then comes back out of idle, it is allowed to re-roll, since it could be a new name resolver instance?

Reply:
Yes, I think that's fine.